css/style.css

```diff
@@ -0,0 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  margin: 0 auto;
+  text-align: center;
+}
+#title {
+  font-size: 30px;
+  margin-top: 40px;
+  margin-bottom: 20px;
+}
+#description {
+  font-style: italic;
+  font-size: 20px;
+  margin-bottom: 20px;
+}
+#survey-form {
+  max-width: 60%;
+  margin: 0 auto;
+  background: rgba(155, 150, 150, 0.5);
+  padding: 50px;
+  border-radius: 10px;
+  text-align: justify;
+  font-size: 20px;
+  color: rgb(0, 0, 0);
+}
+#survey-form input,
+#survey-form textarea,
+#survey-form select {
+  display: block;
+  padding: 0 10px;
+  border: none;
+  outline: none;
+}
+#survey-form input[type="radio"] {
+  display: inline;
+  margin-left: 50px;
+}
+#survey-form input[type="checkbox"] {
+  margin-left: 50px;
+  display: inline-block;
+  height: 15px;
+}
+#survey-form input[type="text"],
+#survey-form input[type="email"],
+#survey-form input[type="number"] {
+  height: 40px;
+  width: 90%;
+  margin: 10px auto;
+}
+#survey-form select {
+  height: 40px;
+  width: 90%;
+  margin: 10px auto;
+}
+#survey-form textarea {
+  width: 90%;
+  margin: 10px auto;
+  padding: 10px;
+}
+#survey-form label {
+  padding-left: 40px;
+}
+#submit {
+  margin: auto;
+  width: 50%;
+  height: 50px;
+  outline: none;
+  border: none;
+  border-radius: 5px;
+  padding: 10px;
+  font-size: 16px;
+  text-transform: uppercase;
+  background-color: #000;
+  opacity: 0.7;
+  color: #fff;
+  cursor: pointer;
+  transition: 0.5s;
+}
+#submit:hover {
+  opacity: 1;
+}

```

Comment:
- In the future it would be best to use classes instead of ids

Reply:
- [ ] [REQUIRED] Please, make sure that you mention the reason for the requested changes so that the student can learn for his mistakes